codegen/golang.go

```diff
@@ -120,31 +195,15 @@ func NewTableFromStruct(name string, obj interface{}, opts ...TableOptions) (*Ta
 
 	for i := 0; i < e.NumField(); i++ {
 		field := e.Type().Field(i)
-		if len(field.Name) == 0 {
-			continue
-		}
-		if unicode.IsLower(rune(field.Name[0])) {
-			continue
-		}
-		if sliceContains(t.skipFields, field.Name) {
```

Comment:
@erezrokah why move all those three checks into the `isUnwrap` ? as those are not really related?

Reply:
I'm not sure I understand, those are extracted to `ignoreField` which is called by `addColumnFromField`.
`shouldUnwrap` is `return isFieldStruct(field.Type) && (t.unwrapAllEmbeddedStructFields && field.Anonymous || sliceContains(t.structFieldsToUnwrap, field.Name))`